core/src/prog/sumo/Player.java

```diff
@@ -104,4 +121,13 @@ public void setPosition(final int newPosition) {
     public Texture getTexture() {
         return texture;
     }
+
+    private void roundOver(Player otherPlayer, int thisPosition,
+                           int otherPosition) {
+
+        //Insert ROUND OVER HERE
```

Comment:
Using the TODO-tag is nice so we can find it faster when doing the [game loop-issue](https://github.com/LiviaValenti/TDT4240-sumo-battle-tap/issues/11) ❤️ 
```suggestion
        // TODO: INCREMENT WHEN ROUND IS OVER
```